cmd/libvirt-provider/app/app.go

```diff
@@ -124,7 +126,10 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 		"constructed from the streaming-address")
 
 	fs.StringVar(&o.Servers.Metrics.Addr, "servers-metrics-address", "", "Address to listen on exposing of metrics. If address isn't set, server is disabled.")
-	fs.DurationVar(&o.Servers.Metrics.GracefulTimeout, "servers-metrics-gracefultimeout", 2*time.Second, "Graceful timeout for shutdown metrics server. Ideally set it little longer as idletimeout.")
+	fs.DurationVar(&o.Servers.Metrics.GracefulTimeout, "servers-metrics-gracefultimeout", 2*time.Second, "Graceful timeout for shutdown metrics server.")
+
+	fs.StringVar(&o.Servers.LibvirtConnect.Addr, "servers-libvirt-connect-address", "127.0.0.1:8080", "Address to listen on health check liveness.")
```

Comment:
it is healtcheck server not libvirtconnect. Only one healthcheck endpoint (liveness probe) can exist and it will be use for multiple check mechanism in future.

Reply:
Changed to healthcheck.